docs/java-rest/high-level/document/multi-term-vectors.asciidoc

```diff
@@ -0,0 +1,58 @@
+--
+:api: multi-term-vectors
+:request: MultiTermVectorsRequest
+:response: MultiTermVectorsResponse
+--
```

Comment:
you can stick things like `:tvrequest: TermVectorsRequest` in here as well. Of course call it what ever you want :)